### PR TITLE
header design changed

### PR DESCRIPTION
**Pull request by msdio**
related to #65 

## 📄 What I've done
image

### ⛱️ Major Changes

- header design changed
- Add image resource in `src > assets > images`

### 🙋 Review Points

- I put images in `src > assets > images`, and used like this:
```tsx
<img src={require('../images/logo.png')} alt='test' />
```
is there any other way to use it better?

- also, I made react component that using image asset. how did you used images and icons in other projects?
- color names will change to constants after setting custom themes

### CODE REVIEW

File: src/components/header/Header.tsx [hunk 1]
```diff
@@ -40,27 +62,24 @@ const Header = () => {
       </RouterLink>
 
       <Stack direction='row' spacing={2} marginLeft='auto'>
-        <Link
-          href='https://github.com/msdio/stackticon'
-          sx={{
-            textDecoration: 'none',
-          }}
-        >
+        <GithubLink href='https://github.com/msdio/stackticon'>
```
Comment (userJu) [inline src/components/header/Header.tsx:65]: I think there were a lot of parts using the stackticon github URL! (Especially in the saving as img / saving in md format!)
How about using this part as a constant?

File: src/components/header/Header.tsx [hunk 2]
```diff
@@ -40,27 +62,24 @@ const Header = () => {
...
-              <path d={Svg.github} />
+              <path d={Svg.github} fill='#0066ff' />
             </SvgIcon>
-            <Typography color='primary.main'>visit github</Typography>
+            <Typography color='#0066ff' fontWeight='600' fontSize='22px'>
+              Visit Github
+            </Typography>
           </Box>
-        </Link>
+        </GithubLink>
```
Comment (userJu) [inline src/components/header/Header.tsx:82]: I think the renamed better specifies that it is a Github Link. 👍👍

**Review by userJu — APPROVED** [review]
Good job HARU! The header that went through the hands of the designer is definitely pretty😖👍

File: src/assets/icons/logoWithBackground.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+      borderRadius='4px'
+      bgcolor='#f3eae1'
+      width={`${width}px`}
+      height={`${height}px`}
+      gap='9.6px'
+    >
+      <img
+        src={require('../images/logo.png')}
+        alt='logo'
+        loading='lazy'
```
[the remark this answers is not in the document]
Comment (msdio) [reply src/assets/icons/logoWithBackground.tsx:23]: I considered this for tiny optimization. I agree with that it doesn't have big importance. I will change it right away 🙂

File: src/components/header/Header.tsx (hunk 1 above)
Comment (msdio) [reply src/components/header/Header.tsx:65]: Good point 🙂

**Pull request merged**